_opensearch/cluster.md

```diff
@@ -12,23 +12,26 @@ OpenSearch can operate as a single-node or multi-node cluster. The steps to conf
 
 To create and deploy an OpenSearch cluster according to your requirements, it’s important to understand how node discovery and cluster formation work and what settings govern them.
 
-There are many ways to design a cluster. The following illustration shows a basic architecture:
+There are many ways to design a cluster. The following illustration shows a basic architecture that includes a four-node cluster that has one dedicated cluster manager node, one dedicated coordinating node, and two data nodes that are cluster manager-eligible and also used for ingesting data.
```

Comment:
Global: Please use "cluster-manager-eligible" as a compound modifier preceding a noun, but do not hyphenate it when it follows a noun. It is a cluster-manager-eligible node, but the node is cluster manager eligible.

Reply:
Thanks @natebower !